src/dataflow/utilities/export.ts

```diff
@@ -0,0 +1,32 @@
+import { DataSequence } from "../components/dataflow-program-graph";
+
+export const exportCSV = (dataSequences: DataSequence[]) => {
+  if (!dataSequences) {
+    return;
+  }
+  if (dataSequences.length > 0) {
+    const headers: string = "timestamp,name,units,value";
+    const rows: string[] = [];
+    dataSequences.forEach((sequence) => {
+      sequence.data.forEach((d) => {
+        // x is the timestamp, y is the value
+        const row: string[] = ["" + d.x, sequence.name, sequence.units, "" + d.y];
```

Comment:
Should we be worried about commas or other special characters in sequence names (seems plausible given that they're user-entered) or units (less likely) here?

Reply:
This is the most interesting question - we should definitely consider some restrictions on names. While Rete seems to permit pretty much anything without adverse effects, there's no need for Bobby Tables to enter his full name here.